src/util/widgethelper.h

```diff
@@ -49,6 +50,9 @@ inline QScreen* getScreen(
 #endif
 }
 
+/// QSize for stretching a list widget attempting to show entire column
+QSize adjustedListWidgetSize(const QListWidget& listWidget, const QWidget& parent);
```

Comment:
Usually functions should be named with verbs in present tense. But using *adjust* would be confusing in this case. Any ideas? Both *calc* or *get* are very generic and don't really improve the situation. Otherwise let's make an exception.

Reply:
I just wanted to describe the return value, not the function.
`adjust` isn't the best choice admittedly. maximizeListWidgetSize() would also work for me.

Reply:
getAdjustedListWidgetSize?

Reply:
actually `maximizeListWidget()`, see review below about encapsulating `setMinimumSize()`

Reply:
`growListWidget()`